site-src/geps/gep-1709.md

```diff
@@ -37,8 +45,295 @@ which implementations can prove they satisfy and be recognized for.
+> an individual, a company, e.t.c.. Organizations can theoretically have more
+> than one `<project>` name and submit separate reports for each of them.
+
+> **NOTE**: The above assumes that the `acmeorg-acme` project was already
+> passing for versions prior to `v0.7.0` and that the new passing version was
+> added to the top of the list. This allows implementations to report their
+> historical passes which will become relevant as we get into future major
+> releases. You can also see that there's been progression to add the
+> `HTTPRouteQueryParamMatching` feature this release.
+
+> **NOTE**: The `maintainers` field indicates the Github usernames or team
+> names of those who are responsible for maintaining this file, so they can be
+> easily contacted when needed (e.g. for relevant release announcements
+> regarding conformance, e.t.c.).
+
+The implementation can compile its `ConformanceReport` and upload it to the
+Gateway API project by creating a pull request. The file should be uploaded
+to the following sub-directory:
+
+```console
+conformance/results/<organization>-<project>.yaml
+```
+
+> **NOTE**: the first time an implementation adds their report, they need
+> to also add the Golang templating to compile that report into their section
+> of the implementations page if that's not already been handled.
+
+This will start the certification process which will result in the
+implementation receiving a badge they can use on their project pages and/or
+repositories which indicates their conformance levels and link to their
+conformance reports in the upstream Gateway API website.
+
+> **NOTE**: No verification process (to prevent intentionally incorrect
+> conformance results) will be implemented at this time. We expect that this wont
+> be an issue in our community and even if someone were to try and "cheat" on
+> the reporting the reputation loss for being caught would make them look very
+> bad and would not be worth it.
+
+#### Certification Process
+
+During the process of the pull request described in the reporting process the
+implementer will be expected to run a command to compile updates before the
+pull request will be allowed to merge:
+
+```console
+make conformance.reports
+```
+
+Any updates will be compiled (via Golang templates) and will then be submitted
+as a commit in the pull request.
+
+The conformance report will emit a heading that displays the latest release
+version which conformance has been reported for, along with a note about any
+newer releases which have yet to be reported.
+
+CI will provide [badges](https://shields.io) to contributors upon completion.
+
+> **NOTE**: Badges which cover `Extended` support (such as `Layer7Extended`)
+> will look the same regardless of whether the implementation only fulfills
+> the extended feature set partially. The badge links will direct to the full
+> report, which will include a list of any feature exceptions.
```

Comment:
maybe a different colour for semi implemented Extended support?

Reply:
I'm open to doing something like this in the future if implementers complain about it... but I feel like extended is a completely reasonable place to have some features but not all.